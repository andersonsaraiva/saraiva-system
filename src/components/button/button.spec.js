import { shallowMount } from '@vue/test-utils';
import Comp from './button.vue';

describe('Button.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Comp);
    expect(wrapper.element).toMatchSnapshot();
  });

  it('does not throw console error when info type', () => {
    const wrapper = shallowMount(Comp);

    expect(wrapper.vm.$options.props.size.validator('bananinha')).toBe(null);
  });
});
