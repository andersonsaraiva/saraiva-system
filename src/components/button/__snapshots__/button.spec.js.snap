// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.vue matches snapshot 1`] = `
<button
  class="on-button"
>
  <!--v-if-->
</button>
`;
